lib/cart.rb

```diff
@@ -0,0 +1,59 @@
+require_relative "menu_item"
+require_relative "menu"
+require_relative "order_item"
+require_relative "customer"
```

Comment:
Good comment, thanks.  I should have slowed down and tested if I could have just used "require".  Regarding your question, my understanding is that `require_relative` is used to load files that are not un Ruby's default load path.  For reference, here's what I found on StackOverflow: 

require_relative complements the builtin method require by allowing you to load a file that is relative to the file containing the require_relative statement.

For example, if you have unit test classes in the "test" directory, and data for them under the test "test/data" directory, then you might use a line like this in a test case:
require_relative "data/customer_data_1"